yb-voyager/src/utils/utils.go

```diff
@@ -256,6 +257,20 @@ func GetSortedKeys(tablesProgressMetadata map[string]*TableProgressMetadata) []s
 	return keys
 }
 
+func RemoveExcludeList(includeList []string, excludeList []string) []string {
+	if len(includeList) == 0 || len(excludeList) == 0 {
+		return includeList
+	}
+	var finalList []string
+	for _, object := range includeList {
+		if slices.Contains(excludeList, object) {
```

Comment:
Does this code handle object types mentioned in various case combinations e.g. `table`, `Table`, or `TABLE`?

Reply:
I will have to add this change in the next patch-there were many test cases I had to cover, seems like I missed this while checking the compatibility with all source DB types

Reply:
there is some function i think `strings.EqualFold()` which can be handy, it does the string comparison without considering the case and all

Reply:
I would then have to introduce another level of indentation by running a nested for loop in order to explicitly compare each string pair-wise 😅 

For this specific functionality, I have ensured case-insensitivity elsewhere